bin/BackupPC_Admin_SCGI

```diff
@@ -130,6 +132,7 @@ BEGIN {
         sleep(60);
         exit(1);
     }
+    # TODO: What should we do with this RSS related stuff ?
```

Comment:
@craigbarratt I'm unsure what to do with this ? I feel like this can safely be removed right ?